src/main.ts

```diff
@@ -118,13 +125,75 @@ actionsToolkit.run(
         core.setOutput('metadata', metadatadt);
       });
     }
+    await core.group(`Reference`, async () => {
+      const ref = await buildRef(toolkit, startedTime, inputs.builder);
+      if (ref) {
+        core.info(ref);
+        stateHelper.setBuildRef(ref);
+      } else {
+        core.warning('No build ref found');
+      }
+    });
+    if (err) {
+      throw err;
+    }
   },
   // post
   async () => {
+    if (stateHelper.buildRef.length > 0) {
+      await core.group(`Generating build summary`, async () => {
+        if (process.env.DOCKER_BUILD_NO_SUMMARY && Util.parseBool(process.env.DOCKER_BUILD_NO_SUMMARY)) {
+          core.info('Summary disabled');
+          return;
+        }
+        if (stateHelper.builder && stateHelper.builder.driver === 'cloud') {
+          core.info('Summary is not yet supported for cloud driver');
```

Comment:
I think we can use the product name here, wdyt?

```suggestion
          core.info('Summary is not yet supported with Docker Build Cloud.');
```